OpenOversight/app/main/views.py

```diff
@@ -636,10 +636,10 @@ def leaderboard():
 def label_data(department_id=None, image_id=None):
     jsloads = ['js/cropper.js', 'js/tagger.js']
     if department_id:
-        department = Department.query.filter_by(id=department_id).one()
+        department = Department.query.filter_by(id=department_id).first()
```

Comment:
Just curious @leifan89, can you elaborate why this change is necessary? Are there situations that an invalid department_id is provided that shouldn't result in an error?

Reply:
Thanks for the review!

Unfortunately I'm having issues replicating the problem that I encountered before with `one()` instead of `first()`... do you think that it would be better to throw an error explicitly instead of just swallowing it? Happy to restore back to the original `first()` 🙂

Reply:
I think leaving it as `one()` makes more sense, since there shouldn't be any cases in which there are multiple departments with the same department_id that would be returned by the query.

Reply:
So I did manage to replicate the original issue that caused me to change it to `first()`...

As it turns out, it was the test that I had added, in which I tested the scenario of an officer with a mismatched department. In that case, since the query with both that department and that officer fails to find anything (the mismatch case), `one()` fails since it expects exactly 1 result, not 0.

Reply:
@dismantl this particular case is fixed and no longer in the code